ElectronicObserver/Window/Tools/ExpeditionRecordViewer/ExpeditionRecordViewer.xaml

```diff
@@ -0,0 +1,246 @@
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:mocks="clr-namespace:ElectronicObserverTypes.Mocks;assembly=ElectronicObserverTypes"
+	xmlns:pager="clr-namespace:ElectronicObserver.Window.Control.Paging"
+	xmlns:ui="http://schemas.modernwpf.com/2019"
+	Title="ExpeditionRecordViewer"
+	Width="800"
+	Height="450"
+	d:DataContext="{d:DesignInstance local:ExpeditionRecordViewerViewModel}"
+	ui:WindowHelper.UseModernWindowStyle="True"
+	x:TypeArguments="local:ExpeditionRecordViewerViewModel"
+	mc:Ignorable="d"
+	>
+	<Window.Resources>
+		<local:ExpeditionRecordViewerBindingProxy x:Key="Proxy" DataContext="{Binding}" />
+		<DataTemplate DataType="{x:Type local:ExpeditionRecordViewModel}">
+			<Grid Margin="2">
+				<Grid.ColumnDefinitions>
+					<ColumnDefinition SharedSizeGroup="ID" />
+					<ColumnDefinition SharedSizeGroup="Result" />
+					<ColumnDefinition Width="Auto" SharedSizeGroup="Material" />
+					<ColumnDefinition SharedSizeGroup="Items" />
+					<ColumnDefinition SharedSizeGroup="Date" />
+					<ColumnDefinition SharedSizeGroup="FleetImage" />
+					<ColumnDefinition SharedSizeGroup="FleetImageButton" />
+					<ColumnDefinition SharedSizeGroup="Extra" />
+				</Grid.ColumnDefinitions>
+
+				<StackPanel
+					Grid.Column="0"
+					Margin="2 0"
+					VerticalAlignment="Center"
+					Orientation="Horizontal"
+					>
+					<TextBlock Text="{Binding DisplayID}" />
+				</StackPanel>
+
+				<TextBlock
+					Grid.Column="1"
+					VerticalAlignment="Center"
+					Text="{Binding ClearResult}"
+					/>
+				<StackPanel
+					Grid.Column="2"
+					Margin="2 0"
+					VerticalAlignment="Center"
+					Orientation="Vertical"
+					>
+					<UniformGrid Columns="4">
+						<commontools:ImageTextControl Image="{StaticResource Resource.Fuel}" Text="{Binding MaterialFuel}" />
+						<commontools:ImageTextControl Image="{StaticResource Resource.Ammo}" Text="{Binding MaterialAmmo}" />
+						<commontools:ImageTextControl Image="{StaticResource Resource.Steel}" Text="{Binding MaterialSteel}" />
+						<commontools:ImageTextControl Image="{StaticResource Resource.Bauxite}" Text="{Binding MaterialBaux}" />
+					</UniformGrid>
+				</StackPanel>
+				<StackPanel
+					Grid.Column="3"
+					Margin="2 0"
+					VerticalAlignment="Center"
+					Orientation="Horizontal"
+					>
+					<TextBlock Text="{Binding ItemOneCount}" />
+					<common:UseItemIcon Type="{Binding ItemOneName}" />
+					<StackPanel.ToolTip>
+						<TextBlock Text="{Binding ItemOneString}" />
+					</StackPanel.ToolTip>
+					<StackPanel.Style>
+						<Style TargetType="StackPanel">
+							<Setter Property="Visibility" Value="Visible" />
+							<Style.Triggers>
+								<DataTrigger Binding="{Binding ItemOneCount}" Value="{x:Null}">
+									<Setter Property="Visibility" Value="Collapsed" />
+								</DataTrigger>
+							</Style.Triggers>
+						</Style>
+					</StackPanel.Style>
+				</StackPanel>
+				<StackPanel
+					Grid.Column="4"
```

Comment:
You can set the `ToolTip` property directly instead of using `StackPanel.ToolTip`. Might need to set the background to transparent too, so that hover will work properly.